spec/fixtures/teenytest/test/normal.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  'Adds two numbers': function () {
+    // assertions
+  }
+}
```

Comment:
Could you update this with an example from their docs, this is probably a good one to go with: https://github.com/testdouble/teenytest/blob/main/example/simple-node/test/lib/dog-test.js

Reply:
For sure! Something like this https://github.com/vim-test/vim-test/pull/705/commits/c329ec5570ade6810175b3976dedfe97b7f3f115?